redash/models/organizations.py

```diff
@@ -32,6 +32,10 @@ def __str__(self):
     def get_by_slug(cls, slug):
         return cls.query.filter(cls.slug == slug).first()
 
+    @classmethod
+    def get_by_id(cls, _id):
+        return cls.query.filter(cls.id == _id).one()
+
```

Comment:
What if we add a custom base model class that has this method (and maybe be other duplicates)?

http://flask-sqlalchemy.pocoo.org/2.3/customizing/#model-class

```python
# in redash/models/base.py
from flask_sqlalchemy import Model

class RedashModel(Model):
     @classmethod
    def get_by_id(cls, _id):
        return cls.query.filter(cls.id == _id).one()

    def save(self, commit=True):
        postgres.session.add(self)
        if commit:
            postgres.session.commit()
        return self

    def delete(self, commit=True):
        postgres.session.delete(self)
        if commit:
            postgres.session.commit()
        return self

# ...

db = RedashSQLAlchemy(
    model_class=RedashModel,
    session_options={
        'expire_on_commit': False
    },
)
```

Reply:
That's a good idea, but a bit out of scope for this PR. Let's keep it in mind for future refactors.

Reply:
Work for me 😀